src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts

```diff
@@ -135,3 +136,15 @@ export async function fetchGenericAssayData(
     const results = await Promise.all(dataPromises);
     return results;
 }
+
+export async function fetchGenericAssayDataByStableIdsAndMolecularIds(
```

Comment:
this doesn't need to be tagged as async as it does not use await insider of it.  it just returns a promise and that e will be awaited or handled by caller